README.md

```diff
@@ -129,3 +128,29 @@ final class Book
     }
 }
 ```
+
+You can also use the `@required` or `@internal` to make it clearer in certain situations that you want the check skipped or that it is used internally. In this situation in Laravel, while the direct call is not present, it can be used as a property when [referencing relationships in Laravel Eloquent models](https://laravel.com/docs/11.x/eloquent-relationships#defining-relationships):
+
+```php
+<?php
+
+class User extends Model
+{
+    // ...
+
+    /**
+     * @internal
+     */
+    public $timestamps = false;
+
+    /**
+     * @required
+     */
+    public function post(): HasOne
+    {
+        return $this->hasOne(Post::class);
+    }
+
+    // ...
+}
+```
```

Comment:
It would be more useful to have a "--skip-annotation" option, so you can customize this to keywords you need to skip.

Reply:
Ah okay, in that case, I'm pretty sure I wont have enough time to look into implementing this in phpstan or whats necessary, and I need this in a project, I could close this and just use my fork. 

Thanks a lot :)